cs-commons/src/main/java/io/cloudslang/content/constants/ExceptionsValues.java

```diff
@@ -0,0 +1,15 @@
+package io.cloudslang.content.constants;
+
+/**
+ * Created by victor on 31.08.2016.
+ */
+public class ExceptionsValues {
+    public static final String EXCEPTION_DELIMITER = ": ";
+    public static final String INVALID_BOOLEAN_VALUE = "The provided boolean string is invalid. It must be 'true' or 'false'";
+    public static final String INVALID_INTEGER_VALUE = "The provided integer string is invalid.";
```

Comment:
More human messages: integer string is a developer concept, prefer something on the lines of "The provided string cannot be converted to an integer value"